lib/hllcounter.cc

```diff
@@ -348,7 +348,7 @@ uint64_t HLLCounter::estimate_cardinality()
 void HLLCounter::add(const std::string &value)
 {
     HashIntoType x = khmer::_hash_murmur(value);
-    uint64_t j = x & (this->m - 1);
+    uint64_t j = x.as_ull() & (this->m - 1);
```

Comment:
`x` is still 64bit because `_hash_murmur` returns 64bit values, but I can backtrack on the calculations here and avoid hardcoding for 64 bits.